Gemfile.lock

```diff
@@ -422,11 +420,7 @@ DEPENDENCIES
   uglifier
   uuidtools
   vcr
-  vegas
   webmock
   whenever
   will_paginate
   yui-compressor
-
-BUNDLED WITH
-   1.10.2
```

Comment:
It's just bundler. It got introduced with 1.10 and now, I think, you get a warning or even an error if your bundler version is older than this.